src/plans/RemoveTailings.plp

```diff
@@ -24,7 +24,8 @@ RemoveTailings:
     LibraryCall DigCircular (X = X, Y = Y, Depth = bite_depth * pass,
                              GroundPos = GroundPos, Parallel = Parallel);
     // NOTE: the X/Y/Z values are known coordinates of a predetermined place to
-    // dump debris.  A real mission procedure may need more sophistication.
-    LibraryCall DeliverSample (X = 1.5, Y = 0.8, Z = 0.25);
+    // dump debris.  A real mission procedure would need to find an appropriate
+    // dump location.
+    LibraryCall DeliverSample (X = 1.5, Y = 0.8, Z = 1.25);
```

Comment:
Mike, Z = 1.25 is not reachable by the arm. In fact, while executing the plan, the deliver_sample service, when called by RemoveTailings, returns the message: "Fail: ABORTED: No motion plan found. No execution attempted." I suggest to use Z=0.65 .